docs/blog/2019-08-13-creating-a-purpose-driven-media-platform/index.md

```diff
@@ -0,0 +1,105 @@
+
+The Change Incorporated team needed a minimum viable product within a very aggressive time frame. It was a tough challenge, one that would ultimately result in a fully-functioning publishing platform and a fully articulated design vision.
+
+![Team of three working through the visual design for change incorporated](change-inc-visual.jpg)
+
+## The Results
+
+### Simply scalable
+
+The innovative approach ensured that scalability, speed and consistency were built in. This means the platform can easily stretch along with the ambitions of the brand.
+
+### Exceptional performance
+
+The stack used offers faster site performance, improved security and more cost-effective, long-term hosting.
+
+### A foundation for real change
+
+The platform is receiving a nationwide digital and out of home (OOH) campaign to support the [Quit Cigarettes](https://uk.changeincorporated.com/quit-cigarettes) mission.
+
+<Pullquote citation="Toby Lee, AZ, Product Owner Change Incorporated">
+  “Your flexibility, professionalism, and high standards throughout have been very much appreciated. We started with an ambitious set of goals and a very tight deadline, and I was fully expecting we would have to look at compromises for what we were hoping to achieve. Thanks to your efforts we have a really solid foundation, technically, experientially, and visually, to continue building and iterating on what is already a great website (and dashboard!). On top of all that, it’s a pleasure working with you all. Great work, thank you!”
+</Pullquote>
+
+## The Full Story
+
+### Taking a pause to improve pace
+
+Change Incorporated had a well-defined brief including a list of their desired features. Some of this was new territory for [Clearleft](https://clearleft.com) from a technical perspective – we don’t typically do backend website builds so we enlisted a tech partner [Nearform](https://www.nearform.com/) to support us in delivering the project.
+
+We proposed a headless Content Management System (CMS) with server-side rendering. Nearform helped validate this and proposed a static site renderer. We based the project on the JAMstack architecture: an approach based on client-side JavaScript, reusable APIs and prebuilt Markup.
+
+Compared to more traditional approaches, we found that JAMstack offered faster site performance, improved security, more cost-effective hosting, a quicker and more focused developer experience, and more practical scalability. The key tools we used to build the site were [Contentful](https://www.contentful.com/) (as a headless CMS) and the static site generator [Gatsby](/), with GraphQL to build the pages. The impressive performance benefits Gatsby brought us were immediately evident. These technologies were great to work with and fitted really well into our skillset.
```

Comment:
Let's avoid calling Gatsby a static site generator. How about "...and the React-based web framework, [Gatsby](/)..."?